README.md

```diff
@@ -9,6 +9,12 @@ Collection of serverless API to enable public access to data surrounding the Bad
 Interactive Swagger Documentation  
 https://docs.badger.finance/
 
+## Docs:
+
+- [Api v2](./docs/api-v2.md)
+- [MFA](./docs/mfa.md)
+- [Health](./docs/health.md)
```

Comment:
let's remove this one - we ended up axing this code we will need to rethink how health was being handled.